app/views/resturants/show.html.erb

```diff
@@ -27,8 +22,13 @@
 
 <p>
   <strong>Max delivery time:</strong>
-  <%= @resturant.max_delivery_time.strftime('%H:%M') unless @resturant.max_delivery_time.blank? %>
+  <% unless @resturant.max_delivery_time.blank? %>
```

Comment:
I wanted to add 'Minutes' to it. would this work?
`@resturant.max_delivery_time + 'Minutes' || 'Not Available'`